api/src/test/java/io/kafbat/ui/ActiveDirectoryLdapTest.java

```diff
@@ -0,0 +1,35 @@
+package io.kafbat.ui;
+
+import static io.kafbat.ui.container.ActiveDirectoryContainer.DOMAIN;
+
+import io.kafbat.ui.container.ActiveDirectoryContainer;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.test.context.ContextConfiguration;
+
+@ContextConfiguration(initializers = {ActiveDirectoryLdapTest.Initializer.class})
+public class ActiveDirectoryLdapTest extends AbstractActiveDirectoryIntegrationTest {
```

Comment:
I feel like we're missing some tests in this test class :)

Reply:
Ah, this is kinda counter-intuitive. Usually `abstract` test classes are used as a test base for setting something up before executing tests. Here, `ActiveDirectoryLdapTest` looks like it doesn't test anything due to the test itself being pulled up to `AbstractActiveDirectoryIntegrationTest` instead.